index.py

```diff
@@ -1,22 +1,41 @@
 import requests
+from requests.exceptions import RequestException
 import json
 import inspect
+from colorama import init, Fore, Style
+init()
 
 from discord import app_commands, Intents, Client, Interaction
 
+# ASCII logo, uses Colorama for coloring 
+# the logo. To clean the code I've used the replace function
+# to color some of the logo.
+logo = f"""
+{Fore.GREEN}                 &(%
+{Fore.CYAN}  %###((((((((((((((((#
+{Fore.CYAN}###(((((((((((  .(((((
+{Fore.CYAN}####(((((  (((( *********
+{Fore.CYAN}####((    (((#************#
+{Fore.CYAN}##************************#
+{Fore.CYAN}##************************#
+{Fore.CYAN}##************************#
+{Fore.CYAN}##************************#
+{Fore.BLUE}&*************************{Fore.RESET}\n
+""".replace("*", f"{Fore.BLUE}*").replace("(", f"{Fore.GREEN}(")
+
 # inspect.cleandoc() is used to remove the indentation from the message
 # when using triple quotes (makes the code much cleaner)
 # Typicly developers woudln't use cleandoc rather they move the text
 # all the way to the left
-print(inspect.cleandoc("""
+print(logo+inspect.cleandoc(f"""
```

Comment:
Since this is already f-string, might as well be easier to use {logo} inside it?

Reply:
Caused formatting bugs last time I tried it.

Reply:
Ahh, that makes sense, just add a space like `logo + inspect` then, otherwise flake8 will kill me lmao